packages/@react-aria/tooltip/src/useTooltipTrigger.ts

```diff
@@ -25,57 +26,90 @@ interface TooltipTriggerAria {
 
 export function useTooltipTrigger(props: TooltipTriggerAriaProps, state: TooltipTriggerState, ref: RefObject<HTMLElement>) : TooltipTriggerAria {
   let {
-    tooltipProps = {} as TooltipProps,
-    triggerProps = {} as TriggerProps,
     isDisabled
   } = props;
 
-  let tooltipId = useId(tooltipProps.id);
-  let triggerId = useId(triggerProps.id);
+  let tooltipId = useId();
+
+  let isHovered = useRef(false);
+  let isFocused = useRef(false);
+
+  let handleShow = () => {
+    if (isHovered.current || isFocused.current) {
```

Comment:
When is this function called where neither of these is true? Trying to understand why these refs are needed.

Reply:
Ah, for show, i really don't need this check
For hide though, I do